### PR TITLE
Removed Azure Portal and Updated FAQs

### PR DESCRIPTION
**Pull request by ByBrianGoodman**
Removed Azure Portal and Updated FAQs

### CODE REVIEW

File: microsoft-365/admin/manage/idle-session-timeout-web-apps.md [hunk 1]
```diff
@@ -140,10 +137,12 @@ The following Microsoft 365 apps are supported.
 
 - Microsoft Purview Compliance Portal
 
-- Azure portal
-
 If you're working on a different web app with the same account, the activity in that web app won't be applied to the idle session timeout.
 
+### I am active in Azure Portal, but I am logged out of other M365 Apps for inactivity.  Why am I logged out?
```
Comment (JohanFreelancer9) [inline microsoft-365/admin/manage/idle-session-timeout-web-apps.md:142]:
```suggestion
### I am active in Azure Portal, but I am logged out of other Microsoft 365 Apps for inactivity.  Why am I logged out?
```

File: microsoft-365/admin/manage/idle-session-timeout-web-apps.md [hunk 2]
```diff
@@ -140,10 +137,12 @@ The following Microsoft 365 apps are supported.
...
+
+Azure Portal supports a similar inactivity feature, but is supported by Azure Portal only. For more information, see [Azure Portal: Signing-Out + Notification](/azure/azure-portal/set-preferences#signing-out--notifications).
```
Comment (JohanFreelancer9) [inline microsoft-365/admin/manage/idle-session-timeout-web-apps.md:144]:
```suggestion
Azure Portal supports a similar inactivity feature that is supported by Azure Portal only. For more information, see [Azure Portal: Signing-Out + Notification](/azure/azure-portal/set-preferences#signing-out--notifications).
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @ByBrianGoodman, kindly commit the suggested changes. Many thanks.

**Review by ByBrianGoodman — COMMENTED** [review]
@JohanFreelancer9 - Changes look good.

**Pull request merged**